dcp-0008/dcp-0008.mediawiki

```diff
@@ -0,0 +1,341 @@
+<pre>
+DCP: 0008
+Title: Explicit Version Upgrades
+Author: Dave Collins <davec@decred.org>
+Status: Defined
+Created: 2021-08-24
+License: CC0-1.0
+License-Code: ISC
+</pre>
+
+==Abstract==
+
+This specifies modifications to Decred transaction and scripting language
+version enforcement to reject all newer versions until a consensus vote
+explicitly enables a given version along with fully defined semantics for it.
+
+==Motivation==
+
+There are two primary goals that motivate the proposed changes:
+
+* Provide an easy, reliable, and efficient method for software and hardware to determine exactly which rules should be applied to given transaction and script versions
```

Comment:
```suggestion
* Provide an easy, reliable, and efficient method for software and hardware to determine exactly which rules should be applied to a given transaction and script versions
```

add a "the" or "a"

Reply:
I'm pretty sure that it is correct as is since removing the adjectives that modify "versions" results in "... applied to given versions."

I went ahead and modified it to remove the word "given" though which I hope clears up any ambiguity.  Let me know if the updated version sounds better or it requires further tweaks.

Reply:
sounds good